src/pudl_archiver/archivers/gridpathratoolkit.py

```diff
@@ -0,0 +1,160 @@
+"""Archive GridPath RA toolkit renewable generation data.
+
+This dataset was produced by Moment Energy Insights (now Sylvan Energy Analytics).
+It is archived from files stored in the private sources.catalyst.coop bucket.
+
+The files in the GCS bucket map to our file names in the following way:
+
+Solar data:
+* gridpath-ra-toolkit/TemporalData/HourlySolar_byProject.zip --> original_solar_capacity.zip
+* gridpath-ra-toolkit/MonteCarlo_Inputs.zip/temporal_data/solar/ --> aggregated_solar_capacity.zip
+* gridpath-ra-toolkit/MonteCarlo_Inputs.zip/temporal_data/solar_syn/ --> aggregated_extended_solar_capacity.zip
+* ??? --> solar_capacity_aggregations.csv
+
+Wind data:
+* gridpath-ra-toolkit/TemporalData/HourlyWind_byProject.zip --> original_wind_capacity.zip
+* gridpath-ra-toolkit/MonteCarlo_Inputs.zip/temporal_data/wind/ --> aggregated_wind_capacity.zip
+* gridpath-ra-toolkit/MonteCarlo_Inputs.zip/temporal_data/wind_syn/ --> aggregated_extended_wind_capacity.zip
+* ??? --> wind_capacity_aggregations.csv
+
+
+Weather data:
+* gridpath-ra-toolkit/TemporalData/DailyWeatherData_cleaned.csv --> daily_weather.csv
```

Comment:
Probably want to either update this to reflect the new file layout, or just remove it since it's now contained in the archiver.